[Japanese] Update Japanese specification to follow up to 2.0.0

ohbarye opened the pull request:
Compared to the original English document, I found that the Japanese one lacks some parts and there are some differences.

This commit makes Japanese specification follow up to https://github.com/semver/semver.org/commit/b5e2e75374611657c0c62e0bcc8d03172d49e3d5.

ohbarye committed:
Update Japanese specification

this commit makes Japanese specification follow up to https://github.com/semver/semver.org/commit/b5e2e75374611657c0c62e0bcc8d03172d49e3d5.

alexandrtovmach commented:
@ohbarye Thanks for contribution. Do you know any native speaker who can review these changes?

ohbarye commented:
> Do you know any native speaker who can review these changes?

@munierujp @haacked @soukouki @gulping @foomin10 
How about you? I saw you've reviewed or contributed to this project in Japanese.

munierujp commented on lang/ja/index.md:
```diff
@@ -106,17 +171,39 @@ FAQ
+
+### SemVer文字列をチェックするために推奨される正規表現（RegEx）はありますか？
+
+　二つあります。一つは名前付きグループをサポートするシステム（PCRE [Perl Compatible Regular Expressions, i.e. Perl, PHP and R]、Python、Go）のための正規表現です。
```
How about translating `i.e.` to `例：` or `〜などの` ?

ohbarye replied:
Sounds nice. Addressed in df34c714e6f82c8225753d884ab1a31c0759cc92.

munierujp replied:
@ohbarye Nice job! Could you update `lang/ja/spec/v2.0.0.md` too?

ohbarye replied:
Oops, I updated both and rebased into 5cfd798. Thanks.

munierujp commented on lang/ja/index.md:
````diff
@@ -106,17 +171,39 @@ FAQ
+^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+```
+
+　もう一つは番号付きキャプチャグループを使った正規表現です（したがって、cg1 = メジャー、cg2 = マイナー、cg3 = パッチ、cg4 = プレリリース、cg5 = ビルドメタデータを意味します）。これはECMA Script（JavaScript）、PCRE（Perl Compatible Regular Expressions, i.e. Perl, PHP and R）、Python、Goにおいて互換性があります。
````
How about translating `i.e.` to `例：` or `〜などの` ?

ohbarye replied:
The same goes with https://github.com/semver/semver.org/pull/243#discussion_r439216499

munierujp replied:
@ohbarye Nice job! Could you update `lang/ja/spec/v2.0.0.md` too?

munierujp reviewed (commented):
Please read my comments.

ohbarye committed:
Translate `i.e.` to Japanese

https://github.com/semver/semver.org/pull/243#discussion_r438849795
https://github.com/semver/semver.org/pull/243#discussion_r438851316

ohbarye commented:
@munierujp Thanks for your review, I've addressed your points!

munierujp reviewed (approved):
It seems good for Japanese.

alexandrtovmach commented:
@munierujp thanks

ohbarye commented:
Thank both of you!
